core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -304,16 +286,18 @@ public String toString() {
    * @return the Schema to project
    */
   private Schema lazyColumnProjection() {
+    Collection<String> selectedColumns = context.selectedColumns();
```

Comment:
This use of `selectedColumns` works because it is a getter.

Reply:
I wonder if it would be valuable to publish (add in repo) a naming style guide to standardize this and save time. What do you think?